### ISSUE TITLE
Support for BFMatcher Constructor for crossCheck=True

### ISSUE THREAD

**Issue by eek** [justadudewhohacks/opencv4nodejs#400]
Hi there!

Is there a way to easily enable crossCheck for Brute Force Feature Matcher?

```# create BFMatcher object
bf = cv2.BFMatcher(cv2.NORM_HAMMING, crossCheck=True)
# Match descriptors.
matches = bf.match(des1,des2)
```

Currently Using the SURF Detector and would be great to have the crossCheck function.

**Comment by justadudewhohacks** [comment]
Descriptor matching is currently implemented [here](https://github.com/justadudewhohacks/opencv4nodejs/blob/master/cc/modules/features2d/descriptorMatching.cc). Maybe there is an easy way to implement the flag as a parameter. In the worst case the Matcher classes would have to be implemented and exposed.

**Comment by justadudewhohacks** [comment]
Your changes have been published to npm with v4.8.0.
